straxen/plugins/events/event_basics_som.py

```diff
@@ -0,0 +1,93 @@
+import strax
+import numpy as np
+
+from straxen.plugins.events.event_basics import EventBasics
+
+export, __all__ = strax.exporter()
+
+
+@export
+class EventBasicsSOM(EventBasics):
```

Comment:
This plugin is inherited from the classic event-basics and only adds in addition the new SOM fields. It will also provide `event_basics` and is used in the SOM context.